### ISSUE TITLE
CITY_RANGED_ATTACK_MODIFIER not working?

### ISSUE THREAD

**Issue by Vyyt02** [LoneGazebo/Community-Patch-DLL#6502]
_1. Mod version (i.e Date - 3/15):

4/7

_2. Mod list (if using Vox Populi only, leave blank):_
 Too many

_3. Error description:_ 

When I change the CITY_RANGED_ATTACK_MODIFIER (sorry, I am not by my computer, maybe I got the wording wrong), the city attack does not change like it used to before. Is it not used anymore? Or is it because I changed the value mid-game? Is there a new way to decrease the RCS of cities?

**Comment by LoneGazebo** [comment]
CITY_RANGED_ATTACK_STRENGTH_MULTIPLIER
It is used, you need to make it negative.

**Comment by Vyyt02** [comment]
Nope, I even deleted the cache and I can still see the same RCS numbers when I reload. Maybe it does work, but unlike before, it is not savegame compatible?

**Comment by LoneGazebo** [comment]
Shouldn't matter.

**Comment by Vyyt02** [comment]
Hmm, before it worked flawlessly. Now it seems it does not apply at all. I tried with a new game, the same result. I even removed the ranged attack from agriculture and started a new game with IGE. I can see the ranged attacked disappeared from agriculture in the tech tree, but the city still shoots on turn 1 and it deals a bit too much damage for my taste.

Maybe I am doing something wrong, I change quite a lot of things in my options file mod-mod (but I don't see any difference in the city RCS even if I change the value directly in VP). Anyway, I don't want to waste your time while you are catching up after a break with the new version. I will think about it later. But would you reopen this issue? If you have time later on, a quick explanation about how the city RCS is actually calculated (maybe on CivFanatics) would be appreciated.

Cheers!

**Comment by Vyyt02** [comment]
I updated VP to 4-17-2, ran the game with VP + IGE + RAS only. A newly found city was dealing 23 damage to a Barb spearman I placed next to it on turn 1.

Then I exited the game to desktop and changed this:
UPDATE Defines SET Value = '0' WHERE Name = 'CITY_RANGED_ATTACK_STRENGTH_MULTIPLIER';

... to this:
UPDATE Defines SET Value = '-200' WHERE Name = 'CITY_RANGED_ATTACK_STRENGTH_MULTIPLIER';

directly in CoreDefines.sql (= not mod-modding). 

I launched the game again and the RCS damage from the city is still 23. I am pretty sure the CITY_RANGED_ATTACK_STRENGTH_MULTIPLIER is
